scipy/signal/windows.py

```diff
@@ -1658,6 +1736,7 @@ def exponential(M, center=None, tau=1., sym=True):
     ('hamming', 'hamm', 'ham'): (hamming, False),
     ('hanning', 'hann', 'han'): (hann, False),
     ('kaiser', 'ksr'): (kaiser, True),
+    ('lanczos', 'lcz'): (lanczos, True),
```

Comment:
That's fair enough, I did not know it was for backwards compatibility. I have removed the spelling 'lcz' in the latest commit.